RandomData.Api/NumberGenerators/Dtos/NumberParameters.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace RandomData.Api.NumberGenerators.Dto
```

Comment:
Name of folder is 'Dto's so namespace should be "RandomData.Api.NumberGenerators.Dtos"